third_party/amd/backend/compiler.py

```diff
@@ -147,6 +147,8 @@ def make_llir(src, metadata, options):
         pm = ir.pass_manager(mod.context)
         pm.enable_debug()
         amd.passes.ttgpuir.add_decompose_unsupported_conversions(pm)
+        lds_size = 65536
```

Comment:
I am not sure, where to place code choosing LDS size, so it is plain constant at this point.
Let's introduce some interface in later PR.

Reply:
It should be convenient to rebase onto Lei's PR https://github.com/openai/triton/pull/3808